lib/ember-cli/app.rb

```diff
@@ -95,10 +95,14 @@ def ember_path
 
     private
 
-    delegate :match_version?, :non_production?, to: Helpers
+    delegate :match_version?, to: Helpers
     delegate :configuration, to: EmberCLI
     delegate :tee_path, :npm_path, :bundler_path, to: :configuration
 
+    def non_production?
+      !Rails.env.production?
```

Comment:
Ok, this is a bigger issue that it seems.

The gem has essentially three modes of operation:
- development
  Starts a build server on first request, runs it until the process dies.
- test
  Builds assets once, does nothing after.
- production
  Does nothing.

I guess instead of having this logic being spread out in lots of different places, we could make a single config option to let user set the operation mode manually.